test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/ObjectExplorer/ObjectExplorerServiceTests.cs

```diff
@@ -120,154 +138,212 @@ private void ExpandAndVerifyDatabaseNode(string databaseName, ObjectExplorerSess
+
         }
 
-       // [Fact]
+        // [Fact]
         public async void VerifySql2016Objects()
         {
-            var query = LoadScript("Sql_2016_Additions.sql");
-            string uri = "VerifySql2016Objects";
-            string databaseName = null;
-
+            var query = LoadScript("Sql_2016_Additions.sql");
+            string uri = "VerifySql2016Objects";
+            string databaseName = null;
+
             using (SqlTestDb testDb = SqlTestDb.CreateNew(TestServerType.OnPrem, false, databaseName, query, uri))
             {
                 var session = await CreateSession(testDb.DatabaseName, uri);
                 var databaseNodeInfo = await ExpandServerNodeAndVerifyDatabaseHierachy(testDb.DatabaseName, session);
-                await ExpandTree(databaseNodeInfo, session);
-                CancelConnection(uri);
+                await FindNodeByLabel(databaseNodeInfo, session, SR.SchemaHierarchy_Tables);
             }
+            CancelConnection(uri);
+
         }
 
-       // [Fact]
+        // [Fact]
         public async void VerifySqlObjects()
         {
-            var query = LoadScript("Sql_Additions.sql");
-            string uri = "VerifySqlObjects";
-            string databaseName = null;
-
+            var query = LoadScript("Sql_Additions.sql");
+            string uri = "VerifySqlObjects";
+            string databaseName = null;
+
             using (SqlTestDb testDb = SqlTestDb.CreateNew(TestServerType.OnPrem, false, databaseName, query, uri))
             {
                 var session = await CreateSession(testDb.DatabaseName, uri);
                 var databaseNodeInfo = await ExpandServerNodeAndVerifyDatabaseHierachy(testDb.DatabaseName, session);
-                await ExpandTree(databaseNodeInfo, session);
-                CancelConnection(uri);
+                await FindNodeByLabel(databaseNodeInfo, session, SR.SchemaHierarchy_Tables);
             }
+            CancelConnection(uri);
+
         }
 
-       // [Fact]
+        // [Fact]
         public async void VerifyFileTableTest()
         {
-            var query = LoadScript("FileTableTest.sql");
-            string uri = "VerifyFileTableTest";
-            string databaseName = null;
-
+            var query = LoadScript("FileTableTest.sql");
+            string uri = "VerifyFileTableTest";
+            string databaseName = null;
+
             using (SqlTestDb testDb = SqlTestDb.CreateNew(TestServerType.OnPrem, false, databaseName, query, uri))
             {
                 var session = await CreateSession(testDb.DatabaseName, uri);
                 var databaseNodeInfo = await ExpandServerNodeAndVerifyDatabaseHierachy(testDb.DatabaseName, session);
-                await ExpandTree(databaseNodeInfo, session);
-                CancelConnection(uri);
+                await FindNodeByLabel(databaseNodeInfo, session, SR.SchemaHierarchy_Tables);
             }
+            CancelConnection(uri);
+
         }
 
         //[Fact]

```

Comment:
Yes I'll enable them when I can figure out why they are crashing when running all together. I'll address your comments in a new PR.